_benchmark/user-guide/understanding-results.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Understanding benchmark results
+nav_order: 22
+parent: User guide
+---
+
+
+At the end of each test run, OpenSearch Benchmark creates a summary of test results metrics like service time, throughput, latency, and more. These metrics provide insights into how the workload selected performed on a benchmarked OpenSearch cluster.
+
+The following guide gives information about how to understand the results of the summary report.
+
+## OpenSearch Benchmark runs
+
+OpenSearch Benchmark runs a series of nightly tests targeting the OpenSearch development cluster. The results of these test runs can be found on https://opensearch.org/benchmarks. The results show the metrics that span different test runs and target both recent and future versions of OpenSearch.
```

Comment:
```suggestion
OpenSearch Benchmark team runs nightly tests targeting the OpenSearch development cluster, comparing throughput and latency numbers for both recent and future versions of OpenSearch. The results of these tests can be found on https://opensearch.org/benchmarks. 
```

Reply:
Let me know what you think of this @hdhalter

Reply:
Yes, good to provide the distinction! Just add 'the'. The OpenSearch Benchmark team runs nightly tests..."